managedtenants/data/selectorsyncset.yaml.j2

```diff
@@ -305,6 +305,11 @@ items:
     pagerdutyApiKeySecretRef:
       name: pagerduty-api-key
       namespace: pagerduty-operator
+    serviceOrchestration:
+      enabled: "false"
+      ruleConfigConfigMapRef:
+        name: osd-serviceorchestration
```

Comment:
Have you verified if the configmap exists in the expected namespace on all the Hive shards?

Reply:
Thanks for the detailed info, really helps!

I'd like to see this appended to the PR description and the Jira comments.

Reply:
Sure. I will update the Jira.